ZelBack/src/services/appsService.js

```diff
@@ -8926,18 +8940,22 @@ async function checkMyAppsAvailability() {
         ip: myIP,
         appname: app.name,
         ports: appPorts,
+        pubKey,
```

Comment:
Should we assign `pubKey` to an object property?
```suggestion
        pubKey: pubKey,
```

Reply:
pubKey, is a shorthand of pubKey: pubKey,